paasta_tools/kubernetes_tools.py

```diff
@@ -1370,6 +1370,10 @@ def get_enable_envoy_readiness_check(
             "check_envoy", system_paasta_config.get_enable_envoy_readiness_check()
         )
 
+    def get_pod_management_policy(self) -> str:
```

Comment:
Do you think we can add the pod management policy variable to `KubernetesDeploymentConfigDict` [here](https://github.com/Yelp/paasta/blob/bdb878a1bee61e19193f85762374a81b9f6dede4/paasta_tools/kubernetes_tools.py#L286)

Reply:
Thanks for pointing it out! Added.